### PR TITLE
Hidden price sizing

### PR DESCRIPTION
**Pull request by benalleng**
## Description

The fixed width for the hidden price stars was not properly determining the expected width of the number "underneath" (note if your cursor is near the edge of the numers it "flashes") this change ensures that the width is actually fixed and since this span uses monospace we can hard code the width based on the width of the 6 `******` characters at `0.6em` each. Ths the total width when hidden is `3.6em`  

This will break down at > 1000.0t sats as the value will be longer than the stars but if you have that much in stacker news god help you (also it will overflow to the right,not really moving anything)

## Checklist

- [x] Are your changes backwards compatible?

- [x] Did you QA this? Could we deploy this straight to production?

- [x] For frontend changes: Tested on mobile?

## Summary by CodeRabbit

- **Bug Fixes**
	- Fixed layout shifts in the wallet summary component by setting a constant width.

### CODE REVIEW

**Comment by huumn** [comment]
This broke the display on `/wallet`. I went another way in a77f778 to fix it. I'll grant an award for it still.

File: components/hidden-wallet-summary.js [hunk 1]
```diff
@@ -1,22 +1,20 @@
...
-  }, [])
+  // prevent layout shifts when hovering by fixing width to initial rendered width of '******'
+  // We can simply multiply 6 by '0.6em' since monopsace has a fixed width for all characters
+  const width = '3.6em'
 
   return (
     <span
-      ref={ref} style={{ width: fixedWidth ? width : undefined }}
-      className='text-monospace' align='right' onPointerEnter={() => setHover(true)} onPointerLeave={() => setHover(false)}
+      ref={ref} style={{ display: 'block', width: fixedWidth ? width : undefined, textAlign: 'right' }}
```
Comment (benalleng) [inline components/hidden-wallet-summary.js:16]: Ahh, didn't catch that this should have been `display: 'inline-block'`

Comment (huumn) [reply components/hidden-wallet-summary.js:16]: I tried that too but then it doesn't work in the navbar! It's a tricky mistress.

Comment (benalleng) [reply components/hidden-wallet-summary.js:16]: hmm couldn't break it like that did but yours works just the same :+1:

**Pull request closed without merging**